source/postcard/src/max_size.rs

```diff
@@ -218,6 +223,16 @@ impl<T: MaxSize> MaxSize for RangeTo<T> {
     const POSTCARD_MAX_SIZE: usize = T::POSTCARD_MAX_SIZE;
 }
 
+impl<T: MaxSize> MaxSize for core::num::Wrapping<T> {
+    const POSTCARD_MAX_SIZE: usize = T::POSTCARD_MAX_SIZE;
+}
+
+#[cfg(feature = "experimental-derive-core-num-saturating")]
+#[cfg_attr(docsrs, doc(cfg(feature = "experimental-derive-core-num-saturating")))]
```

Comment:
Could we remove the `experimental-derive-core-num-saturating` feature, and change this to `all(feature = "core-num-saturating", feature = "experimental-derive")` instead?

Reply:
I'm gunna go ahead and make this change, shout if you hate it :)

Reply:
Done, left the `#[doc(cfg)]` with just `feature = "core-num-saturating"` because this module is only exported under `feature = "experimental-derive"`